src/geometry/Transformation.js

```diff
@@ -9,21 +9,33 @@
  * @example
  *
  * ```js
- * var transformation = new L.Transformation(2, 5, -1, 10),
+ * var transformation = L.transformation(2, 5, -1, 10),
  * 	p = L.point(1, 2),
  * 	p2 = transformation.transform(p), //  L.point(7, 8)
  * 	p3 = transformation.untransform(p2); //  L.point(1, 2)
  * ```
  */
 
 
-// factory new L.Transformation(a: Number, b: Number, c: Number, d: Number)
+// factory new L.Transformation(a: Number|Number[], b?: Number, c?: Number, d?: Number)

```

Comment:
@IvanSanchez I don't know Leafdoc well enough to say if this is the correct way to specify that `a` is now either `Number` or an array of `Number` - does it look ok to you?